util/reggen/fpv_csr.sv.tpl

```diff
@@ -67,91 +140,92 @@ module ${block.name}_csr_assert_fpv import tlul_pkg::*; import ${block.name}_reg
   // to store all the multi-reg within one basefield, we need to shift the `_fpv` value to the
   // correct bits, then compare with read/write exp_data.
 
-  property wr_P(bit [${addr_msb}:0] addr, bit [DWidth-1:0] act_data, bit regen,
-                bit [DWidth-1:0] mask, int lsb);
-    logic [DWidth-1:0] id, exp_data;
+  property wr_P(bit [${addr_msb}:0] addr, bit [TL_DW-1:0] act_data, bit regen,
+                bit [TL_DW-1:0] mask, int lsb);
+    logic [TL_DW-1:0] id, exp_data;
     (device_wr_S(addr), id = h2d.a_source, exp_data = h2d.a_data & a_mask_bit & mask) ##1
         first_match(##[0:$] d2h.d_valid && d2h.d_source == id) |->
         (d2h.d_error || (act_data << lsb) == exp_data || !regen);
   endproperty
 
   // external reg will use one clk cycle to update act_data from external
-  property wr_ext_P(bit [${addr_msb}:0] addr, bit [DWidth-1:0] act_data, bit regen,
-                    bit [DWidth-1:0] mask, int lsb);
-    logic [DWidth-1:0] id, exp_data;
+  property wr_ext_P(bit [${addr_msb}:0] addr, bit [TL_DW-1:0] act_data, bit regen,
+                    bit [TL_DW-1:0] mask, int lsb);
+    logic [TL_DW-1:0] id, exp_data;
     (device_wr_S(addr), id = h2d.a_source, exp_data = h2d.a_data & a_mask_bit & mask) ##1
         first_match(##[0:$] (d2h.d_valid && d2h.d_source == id)) |->
         (d2h.d_error || ($past(act_data) << lsb) == exp_data || !regen);
   endproperty
 
   // W1C register, if write 1 external data will be cleared to 0;
   // if write 0, internal data won't change
-  property w1c_P(bit [${addr_msb}:0] addr, bit [DWidth-1:0] act_data, bit regen,
-                bit [DWidth-1:0] mask, int lsb);
-    logic [DWidth-1:0] id, exp_data;
+  property w1c_P(bit [${addr_msb}:0] addr, bit [TL_DW-1:0] act_data, bit regen,
+                bit [TL_DW-1:0] mask, int lsb);
```

Comment:
nit: alignment

Reply:
Thank you for reviewing the code. These are updated.